client/src/App.tsx

```diff
@@ -102,6 +104,9 @@ function App() {
 
 	const [figures, setFigures] = useState<IFigure[]>([]);
 
+	const fireWorkContainer = useRef(null);
```

Comment:
would be better named something like animationRef or appRef since it's a ref for the app div
fireWorkContainer at first glance makes me think it's only for fireworks